index.html

```diff
@@ -22,21 +22,23 @@
         <div ng-show="loaded" ng-cloak>
             <div ng-repeat="row in user" class="com">
                 <div class="q">
+                    <span class="created_at">{{row.created_at | date:'yyyy-MM-dd'}}</span>
```

Comment:
Please remove it. We don't need to include this. I want to keep it minimal as much as possible.